modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js

```diff
@@ -17,244 +17,302 @@
  */
 
 var palette = new Rickshaw.Color.Palette({scheme: "classic9"});
-var graphMap = {};
 
 function drawGraph_connectedcup(from, to) {
+    $("#y_axis-temperature").html("");
```

Comment:
.empty() is much faster than html() empty. refer this jsperf 

https://jsperf.com/jquery-html-vs-empty-vs-innerhtml/9